src/signalfx_metadata.py

```diff
@@ -1107,19 +1159,34 @@ def send_top():
     top = {}
     for p in psutil.process_iter():
         try:
-            top[p.pid] = [
-                p.username(),  # user
-                get_priority(p.pid),  # priority
-                get_nice(p),  # nice value, numerical
-                p.memory_info_ex()[1] / 1024,  # virtual memory size in kb int
-                p.memory_info_ex()[0] / 1024,  # resident memory size in kb int
-                p.memory_info_ex()[2] / 1024,  # shared memory size in kb int
-                status_map.get(p.status(), "D"),  # process status
-                p.cpu_percent(),  # % cpu, float
-                p.memory_percent(),  # % mem, float
-                to_time(p.cpu_times().system + p.cpu_times().user),  # cpu time
-                get_command(p)  # command
-            ]
+            if sys.platform == 'darwin':
+                top[p.pid] = [
+                    p.username(),  # user
+                    get_priority(p.pid),  # priority
+                    p.nice(),  # nice value, numerical
+                    p.memory_info_ex()[1] / 1024,  # virtual memory size in kb
+                    p.memory_info_ex()[0] / 1024,  # resident memory size in kb
+                    p.memory_info_ex()[2] / 1024,  # shared memory size in kb
+                    status_map.get(p.status(), "D"),  # process status
+                    p.cpu_percent(),  # % cpu, float
+                    p.memory_percent(),  # % mem, float
+                    to_time(p.cpu_times().system + p.cpu_times().user),  # cpu
+                    p.name()  # command
```

Comment:
it is more like ~80% the same (2 out of 11 different).  But I down with the reason so on the way.